.github/workflows/update-labels.yml

```
@@ -19,46 +19,60 @@ jobs:
       - name: Checkout Repository
         uses: actions/checkout@v4
 
+      - name: Debug Workflow Context
+        run: echo "Payload: $(cat $GITHUB_EVENT_PATH)"
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 23: mapping values are not allowed in this context

(syntax-check)